website/content/en/v0.27.0/troubleshooting.md

```diff
@@ -128,6 +128,22 @@ Your security groups are not blocking you from reaching your webhook.
 This is especially relevant if you have used `terraform-eks-module` version `>=18` since that version changed its security
 approach, and now it's much more restrictive.
 
+If this issue happens when installing karpenter for the first time and the provisionners are defined using the `extraObjects` key from the values file, ensure that :
```

Comment:
Can you make this update in the `content/en/preview` doc rather than in a specific version? That is the version that will propagate changes between versions

Reply:
Done